lib/main.js

```diff
@@ -84,53 +84,46 @@ define(['polyglot', 'moment', 'router', 'leaflet', 'gui', 'helper', 'language'],
           return d.target !== undefined;
         });
 
+        nodes.forEach(function (d) {
+          d.neighbours = [];
+        });
+
         links.forEach(function (d) {
           var unknown = (d.source.node === undefined);
           var ids;
+
           if (unknown) {
+            d.target.node.neighbours.push({ id: d.source.id, link: d, incoming: true });
             ids = [d.source.id.replace(/:/g, ''), d.target.node.nodeinfo.node_id];
           } else {
             ids = [d.source.node.nodeinfo.node_id, d.target.node.nodeinfo.node_id];
+            d.source.node.neighbours.push({ node: d.target.node, link: d, incoming: false });
+            d.target.node.neighbours.push({ node: d.source.node, link: d, incoming: true });
+            if (d.type !== 'fastd' && d.type !== 'L2TP') {
```

Comment:
wird erst in Zeile 120 ehemalig 119 gesetzt ...
kann nicht vorher abgefragt werden. (vorher war es in einer zweiten Schleife)